tests/unit/test_parse_manifest.py

```diff
@@ -108,7 +108,7 @@ def _new_file(self, searched, name, match):
 class TestPartialParse(unittest.TestCase):
     def setUp(self) -> None:
         mock_project = MagicMock(RuntimeConfig)
-        mock_project.cli_vars = ""
+        mock_project.cli_vars = {}
```

Comment:
@ChenyuLInx looks like you accidentally initialized this to a str but this should be a dict